addons/account/tests/test_reconciliation.py

```diff
@@ -1444,3 +1445,341 @@ def test_reconciliation_cash_basis02(self):
                 expected['tax_10']
             )
             index += 1
+
+    def test_reconciliation_cash_basis_fx_01(self):
+        """
+        Having issued an invoice at date Nov-21-2018 as:
```

Comment:
For clarity purposes, I would make explicit the names of the columns and the company's currency